lib/config/options.js

```diff
@@ -217,6 +217,17 @@ module.exports = {
                 height: parseInt(size[1])
             };
         }
+    },
+
+    referenceImageAbsence: {
+        default: 'error',
+        map: function(value, config) {
+            if (value !== 'error' && value !== 'warning') {
+                throw new GeminiError('Field "referenceImageAbsence" must be one of the "error" or "warning".' +
+                ' Default value is "error"');
+            }
+            return path.resolve(config.projectRoot, value);
```

Comment:
Why do any path manipulations? Value, returned from `map` will be used instead of original. You should really return only `value` itself here.